lib/erb_lint/linters/rubocop.rb

```diff
@@ -68,15 +79,16 @@ def inspect_content(processed_source, erb_node)
         source = rubocop_processed_source(aligned_source, processed_source.filename)
         return unless source.valid_syntax?
 
-        team = build_team
-        team.inspect_file(source)
-        team.cops.each do |cop|
-          correction_offset = 0
-          cop.offenses.reject(&:disabled?).each do |rubocop_offense|
-            if rubocop_offense.corrected?
-              correction = cop.corrections[correction_offset]
-              correction_offset += 1
-            end
+        activate_team(processed_source, source, offset, code_node, build_team)
+      end
+
+      if ::RuboCop::Version::String.to_f >= 0.87
```

Comment:
Why do these conditions on a class level, instead of inside the method body?

Is this just to avoid repeated `.to_f >= 0.87` at runtime?

Reply:
Yes. More efficient, and I find it cleaner this way; it makes it clear what actions need be different according to the version.

Reply:
I was actually thinking the opposite, because your version duplicates some code, instead of just putting the conditionals around the code that is actually different.

But I don't have a strong opinion on this; either way it's clear what to remove.